output/content/eforms/how.md

```diff
@@ -1,15 +1,5 @@
 # How to use this profile
 
-```{attention}
```

Comment:
Maybe add an admonition here prompting the reader to read the primer first.

Reply:
Done in https://github.com/open-contracting/european-union-support/pull/191/commits/3b921d582215e8cac28ee677642ebcd2e98df609